src/gateways/createBookConfigLoader.ts

```diff
@@ -0,0 +1,40 @@
+import { BOOKS } from '../config';
+import { acceptStatus } from '../helpers/fetch';
+import Sentry, { Severity } from '../helpers/Sentry';
+
+export type BookVersion = typeof BOOKS[0];
```

Comment:
i don't think this needs to be exported

Reply:
It is imported in `src/test/mocks/bookConfigLoader.ts`.

Reply:
you'd be better off setting the mock `getBookVersionFromUUID` to `ReturnType<typeof createBookConfigLoader>['getBookVersionFromUUID']` so that its forced to match the actual type instead of deriving the same type again